kubernetes_state/tests/fixtures/prometheus.txt

```diff
@@ -127,6 +127,19 @@ kube_deployment_status_replicas_updated{deployment="failingtest",namespace="defa
 kube_deployment_status_replicas_updated{deployment="jaundiced-numbat-kube-state-metrics",namespace="default"} 1
 kube_deployment_status_replicas_updated{deployment="kube-dns",namespace="kube-system"} 1
 kube_deployment_status_replicas_updated{deployment="tiller-deploy",namespace="kube-system"} 1
+# HELP kube_deployment_status_condition The current status conditions of a deployment.
+# TYPE kube_deployment_status_condition gauge
```

Comment:
why did you have to add these? there's only present on recent versions?

Reply:
Yes this was added with 1.9.0 https://github.com/kubernetes/kube-state-metrics/commit/8c7cf669e0295da1738a983617eebc81ad18b8f2